controls/cf_serverd.cf

```diff
@@ -44,6 +31,26 @@ body server control
       # Allow connections from nodes which are out-of-sync
       denybadclocks         => "false";
 
+    control_server_allowtlsversion_defined::
+
+      # Minimum required version of TLS. Set to "1.0" if you need clients
+      # running CFEngine in a version lower than 3.7.0 to connect.
+      #
+      # Example:
+      # allowtlsversion => "1.0";
+
+      allowtlsversion => "$(default:def.control_server_allowtlsversion)";
```

Comment:
seeing this in isolation and trying to think as a new user, I wonder if `def.` wouldn't be better as `augment.` or even `config.`. As well as a mention when we have these in places to give someone a clue where to look next: aka Augments in docs.cfengine.com or MPF.md.

Reply:
Well, there is a lot of history around `def`.

* Prior to 3.18.0 Augments would ONLY target `default:def`.
* The augments_control class that bounces components based on data change does so by targeting `default:def.control_<component>.*` and figures out which components need re-started based on that. So, even though we can now leverage variables outside `default:def` I have not strayed for anything that ends up being used in a `control` body.


This PR contained changes to `MPF.md` which is rendered in the docs to highlight how to configure it. Were you looking for something different here?